source/kernel/Cargo.toml

```diff
@@ -80,6 +80,7 @@ path = "../spitebuf"
 
 [dependencies.mnemos-alloc]
 version = "0.1.0"
+path = "../alloc"
```

Comment:
fwiw: I think this is normally addressed by the top level workspace patch. You could instead add this to your pomelo workspace overrides